viewer/package.json

```diff
@@ -28,9 +29,12 @@
   ],
   "dependencies": {
     "@gaia-project/engine": "workspace:../engine",
+    "@types/chart.js": "^2.9.31",
     "bootstrap": "^4.4.1",
     "bootstrap-vue": "^2.12.0",
+    "chart.js": "^2.9.4",
     "lodash": "^4.17.15",
+    "moment": "^2.29.1",
```

Comment:
Is this used anywhere?

Reply:
chart.js threw a runtime error - the new version of chart.js (which is still beta) doesn't require it anymore.

Reply:
We should use the beta version then

https://bundlephobia.com/result?p=moment@2.29.1
https://bundlephobia.com/result?p=chart.js@3.0.0-beta.12

Not only the chart.js beta is much smaller than the 2.9 version, it also does not require moment which is bulky (and outdated, now there are `date-fns` and `luxon`...).

It's also treeshakable, meaning we can only import parts we need and the rest of the code will not be included in the bundle.

https://www.chartjs.org/docs/master/getting-started/v3-migration

I can help in the migration / do the migration if you want

Reply:
would be great if you can help :smile:  - I already tried the beta, but couldn't get it to work.

Reply:
I did most of the changes there: https://github.com/boardgamers/gaia-project/commit/bb40da5a5a5de6b5d631828b9305a0028e799d9f

Some things to fixes with tooltips probably